### PR TITLE
qt: Use nPowTargetSpacing constant

### PR DESCRIPTION
**Pull request by maflcko**

### CODE REVIEW

**Comment by jonasschnelli** [comment]
Thanks for the quick cleanup!
utACK fa4d47814f298523aeb9a6658cc74fc46d316e33

File: src/qt/modaloverlay.h [hunk 1]
```diff
@@ -9,7 +9,7 @@
 #include <QWidget>
 
 //! The required delta of headers to the estimated number of available headers until we show the IBD progress
-static const int REQ_HEADER_HEIGHT_DELTA_SYNC = 24;
+static constexpr int HEADER_HEIGHT_DELTA_SYNC = 24;
```
Comment (jtimon) [inline src/qt/modaloverlay.h]: No big deal since it's just one more line diff, but out of curiosity, why rename the constant?

**Comment by jtimon** [comment]
utACK fa4d478

File: src/qt/modaloverlay.h (hunk 1 above)
Comment (MarcoFalke) [inline src/qt/modaloverlay.h]: It is not like the number represents a *required* delta, it is basically a more or less arbitrarily chosen delta.

**Comment by morcos** [comment]
utACK fa4d478

File: src/qt/modaloverlay.h (hunk 1 above)
Comment (luke-jr) [inline src/qt/modaloverlay.h]: Fix comment?

Comment (MarcoFalke) [inline src/qt/modaloverlay.h]: Maybe `Assume we are syncing headers, should the estimated number of missing headers be larger than that number`?

**Pull request merged**